hunk-with-comment.html

```diff
@@ -0,0 +1,57 @@
+<!--
+@license
+Copyright (c) 2017 Preview-Code. All rights reserved.
+This code may only be used under the BSD style license found in LICENSE.txt
+-->
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="diff-highlighter.html">
+
+
+<dom-module id="hunk-with-comment">
+  <template>
+
+    <style>
+      :host {
+        display: block
+      }
+    </style>
+
+    <diff-highlighter code="{{code}}" before-start="[[beforeStart]]" after-start="[[afterStart]]" can-split="true"></diff-highlighter>
+    <div> This is a comment diff </div>
+  </template>
+
+  <script>
+    Polymer({
+      is: 'hunk-with-comment',
+      listeners: {
+        'split-hunk': '_split'
+      },
+      properties: {
+        code: {
+          type: String,
+          notify: true
+        },
+        beforeStart: {
+          type: Number,
+          notify: true
+        },
+        afterStart: {
+          type: Number,
+          notify: true
+        },
+        indexItem: Number
+      },
+      _split: function (e) {
+        var x = this.code.split('\n');
```

Comment:
On a second look, I think you can move some of the logic from this function to the `_split` in `diff-highligher`. Then this method can be removed entirely.